packages/cli/src/util/dev/server.ts

```diff
@@ -44,36 +44,39 @@ import {
   detectApiExtensions,
   isOfficialRuntime,
 } from '@vercel/fs-detectors';
-import frameworkList from '@vercel/frameworks';
-
-import cmd from '../output/cmd';
-import link from '../output/link';
-import sleep from '../sleep';
-import { Output } from '../output';
-import { relative } from '../path-helpers';
-import { getDistTag } from '../get-dist-tag';
-import getVercelConfigPath from '../config/local-path';
-import { MissingDotenvVarsError } from '../errors-ts';
-import cliPkg from '../pkg';
-import { getVercelDirectory } from '../projects/link';
-import { staticFiles as getFiles } from '../get-files';
-import { validateConfig } from './validate';
-import { devRouter, getRoutesTypes } from './router';
-import getMimeType from './mime-type';
-import { executeBuild, getBuildMatches, shutdownBuilder } from './builder';
-import { generateErrorMessage, generateHttpStatusDescription } from './errors';
+import frameworks from '@vercel/frameworks';
```

Comment:
I've seen elsewhere you do `import { frameworks } from '@vercel/frameworks';`, so for the sake of consistency consider using the same pattern. Plus, you can do `frameworks.map()` below instead of `frameworks.default.map()`.